rpcserver.go

```diff
@@ -1251,3 +1298,62 @@ func marshallSendAssetEvent(
 		return nil, fmt.Errorf("unknown event type: %T", eventInterface)
 	}
 }
+
+// unmarshalScriptKey parses the RPC script key into the native counterpart.
+func unmarshalScriptKey(rpcKey *tarorpc.ScriptKey) (asset.ScriptKey, error) {
+	var (
+		scriptKey asset.ScriptKey
+		err       error
+	)
+	scriptKey.PubKey, err = schnorr.ParsePubKey(rpcKey.PubKey)
```

Comment:
nit: Consider adding a comment above this line, something like: "Attempting to parse public key as a schnorr public key."